docs_nnx/guides/randomness.ipynb

```diff
@@ -318,8 +344,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Splitting Rngs\n",
-    "When interacting with transforms like `vmap` or `pmap` it is often necessary to split the random state such that each replica has its own unique state. This can be done in two way, either by manually splitting a key before passing it to one of the `Rngs` streams, or using the `nnx.split_rngs` decorator which will automatically split the random state of any `RngStream`s found in the inputs of the function and automatically \"lower\" them once the function call ends. `split_rngs` is more convenient as it works nicely with transforms so we'll show an example of that here:"
+    "## Splitting `nnx.Rngs`\n",
```

Comment:
This is not splitting `nnx.Rngs` directly. Maybe say "Splitting RNG keys".

Reply:
```
## Splitting PRNG keys
```

thank you @IvyZX